CustomScript/customscript.py

```diff
@@ -405,6 +417,8 @@ def to_process(file_path, extensions=['.sh', ".py"]):
     for extension in extensions:
         if file_path.endswith(extension):
             return True
+        else:
+            return False
```

Comment:
I don't like this idea. For example, I may have binary data after __DATA__ literal in perl script.